pkg/api/defaults-apiserver.go

```diff
@@ -80,7 +80,7 @@ func (cs *ContainerService) setAPIServerConfig() {
 		defaultAPIServerConfig["--oidc-groups-claim"] = "groups"
 		defaultAPIServerConfig["--oidc-client-id"] = "spn:" + cs.Properties.AADProfile.ServerAppID
 		issuerHost := "sts.windows.net"
-		if helpers.GetCloudTargetEnv(cs.Location) == "AzureChinaCloud" {
+		if helpers.GetTargetEnv(cs.Location, cs.Properties.GetCustomCloudName()) == "AzureChinaCloud" {
```

Comment:
```suggestion
		if helpers.GetTargetEnv(cs) == "AzureChinaCloud" {
```

It would be better to have a single param over here